src/pages/overview/pdf-extract-api/howtos/extract-api.md

```diff
@@ -125,6 +125,12 @@ schema]( ../../../resources/extractJSONOutputSchema2.json)):
 | Encryption permission                 | DISQUALIFIED_ENCRYPTION                                                                                                                                                                             | File permissions do not allow for content extraction.                 |
 | Complex file            | DISQUALIFIED_CMPLX_FILE                                                                                                                                                                                                       | File contents are too complex for content extraction.                 |
 | Bad PDF             | BAD_PDF                                                                                                                                             | Unable to extract content. File is corrupted, malformed or an empty PDF.                 |
+| Invalid file type               | BAD_PDF_FILE_TYPE                                                                                                                                                                                | The input file is not a PDF file                 |
```

Comment:
Add a period to this and the other ones below to match the style you used previously. (To be clear, I mean add a period to the description.)